tools/azure-sdk-tools/tests/integration/scenarios/setup_py_project_def/setup.py

```diff
@@ -29,15 +29,12 @@
 
 with open("README.md", encoding="utf-8") as f:
     readme = f.read()
-with open("CHANGELOG.md", encoding="utf-8") as f:
-    changelog = f.read()
-
 setup(
     name=PACKAGE_NAME,
     version=version,
     include_package_data=True,
     description="Microsoft Azure {} Library for Python".format(PACKAGE_PPRINT_NAME),
-    long_description=readme + "\n\n" + changelog,
```

Comment:
I know that some packages keep the changelog in their [`readme`](https://github.com/Azure/azure-sdk-for-python/blob/41663d6296c3a2d4013e18fb3621f8ef17c462ce/sdk/keyvault/azure-keyvault-keys/pyproject.toml#L39) field (for `pyproject.toml`, but I've seen similar in `setup.py`). Does this change imply that that's a bad practice?

Reply:
Nah. I just didn't want to populate a fixture changelog, and because I added a test case that _actually partially evaluated_ this code, that test crashed due to the non-existence of the changelog.